Jenkinsfile

```diff
@@ -98,7 +99,7 @@ pipeline {
                                                 credentials('brave-builds-github-token-for-pr-builder')
                                                 github('brave/devops', 'https')
                                             }
-                                            branch('master')
+                                            branch('${params.DEVOPS_BRANCH}')
```

Comment:
why not just `branch(params.DEVOPS_BRANCH)`?

Reply:
because it doesn't work, errors with `ERROR: (script, line 26) No such property: params for class: javaposse.jobdsl.dsl.helpers.scm.GitContext`

Reply:
stupid jenkins